config/files/helper-bins/install-omnibus-product

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Copyright:: Copyright 2017 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# This script is based off of https://gist.github.com/cosimo/3760587
+
+set -eou pipefail
+
+function usage() {
+  cat <<DOC
+Usage: ${0##*/} [-P PRODUCT] [-v VERSION] [-c CHANNEL] [-d DOWNLOAD_DIR] -h
+Download VERSION of PRODUCT from CHANNEL to DOWNLOAD_DIR and install it.
+OPTIONS:
+  -h               Show this message.
+  -P PRODUCT       The name of the product you wish to install. (required)
+  -v VERSION       Which version of the product you wish to install. (default: latest)
+  -c CHANNEL       The channel from which you wish to install the product. (default: stable)
+  -d DOWNLOAD_DIR  The directory to which you wish to download the product package. (default: PWD)
+DOC
+}
+
+channel="stable"
+download_dir="$(pwd)"
+product_name=""
+product_version="latest"
+
+while getopts ":c:d:P:v:h" opt
+do
+  case "$opt" in
+    c)
+      channel="$OPTARG"
+      ;;
+    d)
+      download_dir="$OPTARG"
+      ;;
+    P)
+      product_name="$OPTARG"
+      ;;
+    v)
+      product_version="$OPTARG"
+      ;;
+    h)
+      usage
+      exit 0
+      ;;
+    \?)
+      usage
+      exit 1
+      ;;
+    *)
+      ;;
+  esac
+done
+shift $((OPTIND -1))
+
+# Exit if product_name is not set
+product_name=${product_name:?A product name must be specified}
+
+echo "--- Installing $channel $product_name $product_version"
+download_url="$(mixlib-install download --url --channel "$channel" "$product_name" --version "$product_version")"
+
+mixlib-install install-script | sudo bash -s -- -d "$download_dir" -l "$download_url" 2>&1
```

Comment:
mixlib-install is not part of the toolchain. if we want this to work correctly we need to add it as a base dependency.

Reply:
okay, addressed by adding `omnibus-helper-gems`. Based the list off of what we had in our release engineering repo (for the omnibus ami installation). 

Not sure if there is anything else that needs to be done to make things work.

Reply:
@tduffield thanks for installing the gems. Can you please make this `install-omnibus-product` script use something like `../embedded/bin/mixlib-install`? Using the relative path should allow it to work for both `omnibus-toolchain` and `angry-omnibus-toolchain`.